src/components/dashboard/ColoredSection.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './ColoredSection.scss'
+
+class ColoredSection extends React.Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <div styleName="root">
+        { this.props.head == null ? null : (
```

Comment:
use strict comparison `===` everywhere, 
also use pattern
`this.props.head && <some>`
or 
```javascript
this.props.head && (
  <some>
    ...
  </some>
)```